### PR TITLE
PMM-5695 Fix error messages.

### PR DESCRIPTION
**Pull request by BupycHuk**

### CODE REVIEW

File: models/service_helpers.go [hunk 1]
```diff
@@ -55,6 +55,27 @@ func checkServiceUniqueName(q *reform.Querier, name string) error {
 	}
 }
 
+func validateDBConnectionOptions(socket, host *string, port *uint16) error {
+	if host == nil && socket == nil {
+		return status.Error(codes.InvalidArgument, "Neither socket nor address passed")
```
Comment (askomorokhov) [inline models/service_helpers.go:60]: Do we need a dot at the end of the sentence?

File: models/service_helpers.go [hunk 2]
```diff
@@ -55,6 +55,27 @@ func checkServiceUniqueName(q *reform.Querier, name string) error {
...
+	}
+
+	if host != nil {
+		if socket != nil {
+			return status.Error(codes.InvalidArgument, "Socket and address cannot be specified together")
+		}
+
+		if port == nil {
+			return status.Errorf(codes.InvalidArgument, "Invalid field Port: value '%d' must be greater than '0'", port)
```
Comment (askomorokhov) [inline models/service_helpers.go:69]: Do we need to inherit this error message from the API?
Or need to change it to our own  - more human-readable?

File: models/service_helpers.go [hunk 3]
```diff
@@ -55,6 +55,27 @@ func checkServiceUniqueName(q *reform.Querier, name string) error {
...
+		}
+	}
```
Comment (askomorokhov) [inline models/service_helpers.go:71]: Do we need to validate port range here?

File: models/service_helpers.go (hunk 1 above)
Comment (percona-csalguero) [inline models/service_helpers.go:60]: Error messages shouldn't be capitalized

File: models/service_helpers.go [hunk 4]
```diff
@@ -55,6 +55,27 @@ func checkServiceUniqueName(q *reform.Querier, name string) error {
...
 }
 
+func validateDBConnectionOptions(socket, host *string, port *uint16) error {
+	if host == nil && socket == nil {
+		return status.Error(codes.InvalidArgument, "Neither socket nor address passed")
+	}
+
+	if host != nil {
+		if socket != nil {
+			return status.Error(codes.InvalidArgument, "Socket and address cannot be specified together")
```
Comment (percona-csalguero) [inline models/service_helpers.go:65]: Same here. Error messages shouldn't be capitalized.

File: models/service_helpers.go (hunk 1 above)
Comment (AlekSi) [reply models/service_helpers.go:60]: They should be for gRPC errors: https://jira.percona.com/browse/PMM-4695

File: models/service_helpers.go (hunk 3 above)
Comment (BupycHuk) [reply models/service_helpers.go:71]: No, because uint16 range is between 0 and 65535

File: services/inventory/services_test.go [hunk 5]
```diff
@@ -154,7 +154,7 @@ func TestServices(t *testing.T) {
 			Address:     pointer.ToString("127.0.0.1"),
 			Socket:      pointer.ToString("/var/run/mysqld/mysqld.sock"),
 		})
-		require.EqualError(t, err, "rpc error: code = InvalidArgument desc = setting both address and socket in once is disallowed")
+		require.EqualError(t, err, "rpc error: code = InvalidArgument desc = Socket and address cannot be specified together.")
```
Comment (AlekSi) [inline services/inventory/services_test.go:157]: I wonder why we don't use `tests.AssertGRPCError` there.

File: models/service_helpers.go (hunk 3 above)
Comment (AlekSi) [reply models/service_helpers.go:71]: In theory, we could check that port is not 0 (`pointer.GetUint16(port) == 0`), but I think it is not possible there.

File: models/service_helpers.go [hunk 6]
```diff
@@ -55,6 +55,27 @@ func checkServiceUniqueName(q *reform.Querier, name string) error {
 	}
 }
 
+func validateDBConnectionOptions(socket, host *string, port *uint16) error {
+	if host == nil && socket == nil {
+		return status.Error(codes.InvalidArgument, "Neither socket nor address passed.")
+	}
+
+	if host != nil {
```
[the remark this answers is not in the document]
Comment (BupycHuk) [reply models/service_helpers.go:63]: no, it will ruin all our logic

**Pull request merged**